core/src/main/java/com/amplitude/core/Amplitude.kt

```diff
@@ -205,6 +237,15 @@ open class Amplitude internal constructor(
         return this
     }
 
+    /**
+     * Get the device id.
+     *
+     * @return Device id.
+     */
+    fun getDeviceId(): String? {
```

Comment:
Note for later: Could the logic in `setDeviceId()` cause a bug in setting the deviceId? Should it check the return of await? E.g.
```
if (isBuilt.await()) {
idContainer.identityManager.editIdentity().setDeviceId(deviceId).commit()
}
```

Reply:
@justin-fiedler Currently `build()` (and `isBuilt.await()`) always returns `true`
https://github.com/amplitude/Amplitude-Kotlin/blob/main/core/src/main/java/com/amplitude/core/Amplitude.kt#L108